cli/exceptions.py

```diff
@@ -0,0 +1,35 @@
+class ConfigError(Exception):
```

Comment:
Good approach +1. But can we have a general class for all the exceptions? Or the idea is to club only those exceptions which falls under the scope of the module it is placed?